### PR TITLE
Landing 107 create pop up notification for added cart

### PR DESCRIPTION
**Pull request by mehmetnuricavdar**
# Description: FE/ PopUp modal menu is created for when a movie added to cart

# Jira task link:
https://hackyourfuture-dk.atlassian.net/browse/LANDING-107?atlOrigin=eyJpIjoiZTU1MjUxMWU4YzY3NDZhMGE2YTdhYTBkYzdiMjc3YTgiLCJwIjoiaiJ9

https://hackyourfuture-dk.atlassian.net/browse/LANDING-99?atlOrigin=eyJpIjoiNjRlZmM4ZGE1ZjE4NDhhZWE2MDU0NTJkMjlkNzQ2YmEiLCJwIjoiaiJ9

# Screenshots (if applicable):

https://github.com/HackYourFuture-CPH/CINEMANIA/assets/113375790/64afa09d-ea4e-4667-91ce-b86955d5254a

### CODE REVIEW

File: packages/client/src/components/BigMovieCard/BigMovieCard.jsx [hunk 1]
```diff
@@ -20,14 +20,16 @@ import styled from '@emotion/styled';
...
+import { PopUpModulMenu } from '../OrderReview/PopUpModulMenu';
 
 export const BigMovieCard = ({ currentMovie, currentReview, user }) => {
   const [open, setOpen] = useState(false);
   const [favorites, toggleFavorite] = useFavorites([]);
 
-  const isFavorite = favorites
-    ? favorites.find((favoriteMovie) => favoriteMovie.id === currentMovie.id)
-    : false;
+  const isFavorite =
```
Comment (shpomp) [inline packages/client/src/components/BigMovieCard/BigMovieCard.jsx:29]: 👽

File: packages/client/src/components/BigMovieCard/BigMovieCard.jsx [hunk 2]
```diff
@@ -250,6 +254,7 @@ export const BigMovieCard = ({ currentMovie, currentReview, user }) => {
           />
         </CardContent>
       </Card>
+      <PopUpModulMenu currentMovie={currentMovie} />
```
Comment (shpomp) [inline packages/client/src/components/BigMovieCard/BigMovieCard.jsx:257]: Maybe AddToCartConfirmation or something like that would be better? But not necessarily :D

**Review by shpomp — APPROVED** [review]
🤯  very nice! 👏🏽

File: packages/client/src/components/BigMovieCard/BigMovieCard.jsx (hunk 1 above)
Comment (mehmetnuricavdar) [reply packages/client/src/components/BigMovieCard/BigMovieCard.jsx:29]: i will learn soon :D

File: packages/client/src/components/BigMovieCard/BigMovieCard.jsx (hunk 2 above)
Comment (mehmetnuricavdar) [reply packages/client/src/components/BigMovieCard/BigMovieCard.jsx:257]: thats a good suggestion!

**Pull request merged**